### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akeller**
As discussed in the Camunda 8 Stakeholder meeting - small changes to clarify what's in the Camunda Platform repo as well as some other small tweaks.

### CODE REVIEW

**Review by felix-mueller — APPROVED** [review]
lgtm

**Comment by akeller** [comment]
@christinaausley no urgency here for a review and no need to review the whole README, just the changes.

File: profile/README.md [hunk 1]
```diff
@@ -21,16 +21,19 @@ Camunda Platform 8 is comprised of the following components:
 * Web Modeler - Collaborate and model processes, deploy and start new instances all without leaving Camunda Platform 8.
```
Comment (christinaausley) [inline profile/README.md:21]: Is there a reason why we only link to some of these? Should we link to the documentation for the others?

**Review by christinaausley — APPROVED** [review]
One non-blocking comment 👍

File: profile/README.md (hunk 1 above)
Comment (akeller) [reply profile/README.md:21]: Public vs. Private/Internal repos. The links are only to publicly available repos.

**Pull request merged**